@navikt/core/react/src/button/Button.tsx

```diff
@@ -113,41 +91,32 @@ export const Button: OverridableComponent<ButtonProps, HTMLButtonElement> =
         <Component
           {...(Component !== "button" ? { role: "button" } : {})}
           {...filterProps}
-          ref={mergedRef}
+          ref={ref}
           onKeyUp={composeEventHandlers(filterProps.onKeyUp, handleKeyUp)}
```

Comment:
Should we get `onKeyUp` from destructuring of the `props`, so it is not added through `filteredProps` and then overridden here?

Reply:
Makes sense